scripts/db_restore.sh

```diff
@@ -72,13 +74,50 @@ fi
   set -e
 } | tee >/dev/null
 
-# The first time we run it, we will trigger FK constraints errors
-set +e
-pg_restore -U postgres -h localhost --no-acl --no-owner --role=${LOCALDBUSER} -n public -O -c -d "${LOCALDBNAME}" "${DBDUMP_FILE}" 2>/dev/null
-set -e
+PG_RESTORE_OPTIONS=(-U postgres -h localhost --no-acl --no-owner --role="${LOCALDBUSER}" -n public -O -d "${LOCALDBNAME}")
+PG_RESTORE_COMMAND=pg_restore
+if [ -n "$USE_DOCKER" ]; then
+  PG_RESTORE_COMMAND="sudo docker run --rm --network host -v $(dirname "$DBDUMP_FILE"):/backup postgres:14 pg_restore -h localhost -p 5432 -U opencollective"
+  DBDUMP_FILE="/backup/$(basename "$DBDUMP_FILE")"
+fi
```

Comment:
Added this small thing to easily change `pg_dump`/`pg_restore` versions using docker, but my tests so far surface that their versions are retro compatible, so using the local one (in my case, v16) should give the best results